.github/workflows/release-checklist-comment.yml

```
@@ -6,7 +8,10 @@ on:
       - main
 
 jobs:
-  call-release-checklist-workflow:
+  call-release-workflow:
```

Should this actually be `call-release-checklist-workflow`? That's what I see documented at https://github.com/ASFHyP3/actions#reusable-release-checklist-commentyml
```suggestion
  call-release-checklist-workflow:
```

---

Thanks for the update!